### PR TITLE
fix zh version top blank

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/istio/istio.io/issues/15819

### CODE REVIEW

**Review by windsonsea — APPROVED** [review]

File: content/zh/_index.html [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<!-- TODO: Replace placeholders -->
```
Comment (craigbox) [inline content/zh/_index.html:1]: Wonderful catch :)

Comment (Arhell) [reply content/zh/_index.html:1]: Thanks)

**Comment by wilsonwu** [comment]
Seems the problem still there:

I compared the css class of head section between ZH and EN page, but no problem found :(

**Comment by Arhell** [comment]
> Seems the problem still there? Hope the latest site is up to date: image
> 
> I compared the css class of head section between ZH and EN page, but no problem found :(

This is not css problem
In preview all good
https://deploy-preview-15824--preliminary-istio.netlify.app/latest/zh

**Comment by wilsonwu** [comment]
> > Seems the problem still there? Hope the latest site is up to date: image
> > I compared the css class of head section between ZH and EN page, but no problem found :(
> 
> This is not css problem In preview all good https://deploy-preview-15824--preliminary-istio.netlify.app/latest/zh

Great, thanks so much, not sure when the prod site will be updated.

**Comment by Arhell** [comment]
I think so, because I checked the new PR and the old ones, the problem disappeared

New 
https://github.com/istio/istio.io/pull/15828
https://deploy-preview-15828--preliminary-istio.netlify.app/latest/zh

Old
https://github.com/istio/istio.io/pull/15772
https://deploy-preview-15772--istio.netlify.app/latest/zh/

**Pull request merged**